### PR TITLE
Update firefox_cloud_services_tos.md

### PR DESCRIPTION
**Pull request by mafeld**
Update Firefox Accounts to Mozilla Accounts

### CODE REVIEW

**Comment by mafeld** [comment]
In addition to the changes in the document, we should probably change the title of this doc. @pmac - Is there anything we need to prepare for Bedrock before changing the name of this doc to Mozilla_Accounts_tos?

**Comment by pmac** [comment]
Yes there is actually. We load and show these docs based on the file name. I think the best way to do this is to publish this doc, then make a copy of it with the new name as well. Then we can make the code change for the site. Once that is deployed we can remove the old doc file name. Does that make sense?

The annoying thing is that we'll need to do that for all of the translations as well.

**Comment by mafeld** [comment]
No problem. Let's get to Nov 1, and then we can worry about renaming as clean up afterwards.

File: en/firefox_cloud_services_tos.md [hunk 1]
```diff
@@ -1,28 +1,28 @@
...
+    In order to use some of the Services, you'll need to create a Mozilla account. During registration, you will be asked to set a password. You are responsible for keeping your password confidential and for the activity that happens through your Mozilla account. Mozilla is not responsible for any losses arising out of unauthorized use of your Mozilla account.
 
 3. #### Services 
 
-    (a) Mozilla VPN is a Virtual Private Network in partnership with [Mullvad](https://mullvad.net/). A Firefox account is required.
+    (a) Mozilla VPN is a Virtual Private Network in partnership with [Mullvad](https://mullvad.net/). A Mozilla account is required.
     
     (b) Firefox Relay allows you to keep your primary email address safe and private from online services by making unique, alias email addresses that allow you to send and receive emails without revealing your primary email address. A Firefox account is required.
```
Comment (peiying2) [inline en/firefox_cloud_services_tos.md:27]: Firefox account

**Review by peiying2 — CHANGES_REQUESTED** [review]
The updates look good. However, there are still plenty mentions of Firefox account and Firefox Monitor throughout the document. Please have another go.

File: en/firefox_cloud_services_tos.md (hunk 1 above)
Comment (mafeld) [reply en/firefox_cloud_services_tos.md]: Ok. I think I caught the rest

File: en/firefox_cloud_services_tos.md [hunk 2]
```diff
@@ -1,42 +1,42 @@
...
     
-    (b) Firefox Relay allows you to keep your primary email address safe and private from online services by making unique, alias email addresses that allow you to send and receive emails without revealing your primary email address. A Firefox account is required.
+    (b) Firefox Relay allows you to keep your primary email address safe and private from online services by making unique, alias email addresses that allow you to send and receive emails without revealing your primary email address. A Mozilla account is required.
     
-    (c) Firefox Monitor is an informational service to promote online security by informing you of public data breaches. You can scan email addresses on our website to see most publicly known breaches. If you subscribe with a Firefox account, you will receive: 
+    (c) Firefox Monitor is an informational service to promote online security by informing you of public data breaches. You can scan email addresses on our website to see most publicly known breaches. If you subscribe with a Mozilla account, you will receive:
```
Comment (peiying2) [inline en/firefox_cloud_services_tos.md:29]: Firefox Monitor -> Mozilla Monitor

File: en/firefox_cloud_services_tos.md [hunk 3]
```diff
@@ -1,42 +1,42 @@
...
+    (c) Firefox Monitor is an informational service to promote online security by informing you of public data breaches. You can scan email addresses on our website to see most publicly known breaches. If you subscribe with a Mozilla account, you will receive: 
 
     * Full Report: This includes a complete list of publicly known hacks, including breaches of sensitive websites, such as those for children or adult and dating sites (these sites are excluded from website scans in order to protect the privacy of those individuals). 
     * Breach Alerts: This notifies you if your email address appears in a new data breach. 
     * Safety Tips: This sends you occasional online safety tips from Mozilla.
 
     It is your responsibility to protect your online accounts using strong and different passwords, security questions, two-factor authentication, password managers, and other safety measures. Although Firefox Monitor and [Have I Been Pwned?](https://haveibeenpwned.com/) strive to have current data, there is no guarantee that either will have complete or timely records of all public data breaches.
```
Comment (peiying2) [inline en/firefox_cloud_services_tos.md:35]: Firefox Monitor -> Mozilla Monitor

**Review by peiying2 — CHANGES_REQUESTED** [review]
@mafeld need to change to Mozilla Monitor.

**Comment by mafeld** [comment]
The Mozilla Monitor renaming is happening at a different time, so we are leaving it as Firefox Monitor for this revision.

**Review by peiying2 — APPROVED** [review]

**Pull request merged**